docs/index.rst

```diff
@@ -32,55 +32,89 @@ Next you need to initialize the Raven client and configure it to use your `Sentr
+At this point, Raven is set up to capture and report any uncaught exceptions.
+
+You can optionally pass an object of configuration options as the 2nd argument to `Raven.config`. For
 more information, see :doc:`config`.
 
 Reporting Errors
 ----------------
 
-To get started passing errors to Raven, it is recommended to initialize Raven's global error handler using
-``patchGlobal``. This will cause any uncaught exception which would bubble up to the Node runtime to be
-captured and processed by Raven.
+Raven's ``install`` method sets up a global handler to automatically capture any uncaught exceptions. You can also report errors manually with ``try...catch`` and
+a call to ``captureException``:
 
 .. code-block:: javascript
 
-  client.patchGlobal();
+    try {
+        doSomething(a[0]);
+    } catch (e) {
+        Raven.captureException(e);
+    }
 
-Additionally, you can manually capture and report potentially problematic code with ``try...catch`` and
- ``captureException``:
+You can also use ``wrap`` and ``context`` to have Raven wrap a function and automatically capture any exceptions it throws:
 
 .. code-block:: javascript
+  Raven.context(function () {
+    doSomething(a[0]);
+  });
 
-    try {
-        doSomething(a[0])
-    } catch(e) {
-        client.captureException(e)
-    }
-
-The ``captureException`` method optionally takes an object of configuration options as the 2nd argument. For more information, and to learn about other methods provided by the Raven API, see :doc:`usage`.
+For more information on reporting errors, see :doc:`usage`.
 
 Adding Context
 --------------
 
-While a user is logged in, you can tell Sentry to associate errors with
-user data.  This data is then submitted with each error which allows you
-to figure out which users are affected.
+Code run via ``wrap`` or ``context`` has an associated set of context data, and Raven provides methods for managing that data.
+
+You'll most commonly use this to associate the current user with an exception:
 
 .. code-block:: javascript
 
-    client.setUserContext({
+  Raven.context(function () {
+    Raven.setContext({
+      user: {
         email: 'matt@example.com',
         id: '123'
-    })
+      }
+    });
+    // errors thrown here will be associated with matt
+  });
+  // errors thrown here will not be associated with matt
 
-If at any point, the user becomes unauthenticated, you can call
-``client.setUserContext()`` with no arguments to remove their data.
+This can also be used to set ``tags`` and ``extra`` keys for associated tags and extra data.
 
-Other similar methods are ``client.setExtraContext`` and
-``client.setTagsContext``.  See :ref:`raven-node-additional-context` for more info.
+We can update our context data with ``mergeContext`` or retrieve it with ``getContext``. When an exception is captured by a wrapper, the current context state will be passed as options to ``captureException``.
```

Comment:
Let's not use "we" here – it should be "you" and "your". There are other places where this is inconsistent (e.g. the CORS paragraph in raven-js docs), and we should fix those too.